thingsboard_gateway/connectors/mqtt/mqtt_converter_utils.py

```diff
@@ -0,0 +1,69 @@
+#     Copyright 2022. ThingsBoard
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#
+
+from logging import getLogger
+from re import search
+from simplejson import dumps
+
+from thingsboard_gateway.tb_utility.tb_utility import TBUtility
+
+log = getLogger("service")
+
+
+class MqttConverterUtils:
+
+    @staticmethod
+    def create_result_item(key, value, timestamp):
```

Comment:
Move these methods to `json_mqtt_uplink_converter` as `staticmethods`.

Reply:
Done as discussed